spot-client/src/common/remote-control/remoteControlServer.js

```diff
@@ -272,7 +275,7 @@ export class RemoteControlServer extends BaseRemoteControlService {
      * @returns {Promise} - A Promise resolved when the code has been refreshed.
      */
     refreshJoinCodeWithXmpp() {
-        const roomLock = generateRandomString(3);
+        const roomLock = generateRandomString(3).toLowerCase();
```

Comment:
I don't think the passcode needs to be lowercase, does it?

Reply:
It's not that the passcode needs to be lowercase, but when you enter it on the remote, even if the ui shows them as upper case it does a toLowerCase() and then sends it to the server.
https://github.com/jitsi/jitsi-meet-spot/blob/7f340e04c98710291338838e160b611913cd857a/spot-client/src/spot-remote/ui/views/join-code-entry.js#L310